docs/Development/Creating_a_Repository_guide.md

```diff
@@ -0,0 +1,60 @@
+# GIT REPOSITORY HOW-TO GUIDE
+
+## git init/ git clone
+
+A *[git repository](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository)* is a virtual storage of your project, allowing to save versions of your code and to access when needed.
+
+## SETTING UP A REPOSITORY (Repo) UNDER THE GIT VERSION CONTROL
+
+[How to set-up Git on your device](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup)
+
+### Initializing a new repository: **git init**
+
+To create a new repo, you'll use the **git init** command. git init is a one-time command you use during the initial setup of a new repo. Executing this command will create a new **.git**   subdirectory in your current working directory. This will also create a new branch called master.
```

Comment:
When using terminal commands in content, better show it between three back quotes (`)
Like this:
```zsh
git init
```